source/localizable/components/wrapping-content-in-a-component.md

```diff
@@ -93,6 +93,14 @@ Once yielded the data can be accessed within wrapped content by referencing the
 Finally we want to share the model of the data a user fills out for the post within our `blog-post` and body components.
 To share the `postData` object with the new body component, you can add arguments to the component helper.
```

Comment:
"Finally, we want to share the the blog text that a user fills out for the post within our `blog-post` and body components. To share the blog text with the new body component, we'll add a `postData` argument to the component helper."